projects/storefrontapp/src/app/app.module.ts

```diff
@@ -50,6 +50,7 @@ if (!environment.production) {
 
       // custom routing configuration for e2e testing
       routing: {
+        protected: true,
```

Comment:
If CPQ requires user login, can we move this `protected` into CPQ lib?

Reply:
Yes, we will revert this before merge (first need to prepare our team CI)

Reply:
Done